core/src/main/java/com/sequenceiq/cloudbreak/job/stackpatcher/ExistingStackPatcherJob.java

```diff
@@ -103,18 +106,21 @@ private void unscheduleJob(JobExecutionContext context) {
                 .findFirst();
     }
 
-    private void applyStackPatch(ExistingStackPatchService existingStackPatchService, Stack stack) throws JobExecutionException {
+    private boolean applyStackPatch(ExistingStackPatchService existingStackPatchService, Stack stack) throws JobExecutionException {
         StackPatchType stackPatchType = existingStackPatchService.getStackPatchType();
         if (!existingStackPatchService.isStackAlreadyFixed(stack)) {
             try {
                 if (existingStackPatchService.isAffected(stack)) {
                     LOGGER.debug("Stack {} needs patch for {}", stack.getResourceCrn(), stackPatchType);
                     stackPatchUsageReporterService.reportAffected(stack, stackPatchType);
-                    existingStackPatchService.apply(stack);
-                    LOGGER.info("Stack {} was patched successfully for {}", stack.getResourceCrn(), stackPatchType);
-                    stackPatchUsageReporterService.reportSuccess(stack, stackPatchType);
+                    boolean success = existingStackPatchService.apply(stack);
+                    if (success) {
+                        stackPatchUsageReporterService.reportSuccess(stack, stackPatchType);
+                    }
+                    return success;
```

Comment:
some logs around here please

Reply:
existingStackPatchService.apply logs based on the result, this is why I did not duplicate it there

Reply:
I don't see any logs in `stackPatchUsageReporterService.reportSuccess` so it definitely would be helpful to see if it was called or not

Reply:
I see, then I will add a log to StackPatchUsageReporterService.sendUsageReport